bluesky/plans.py

```diff
@@ -913,17 +929,31 @@ def inner_rel_grid_scan():
 
 
 def relative_inner_product_scan(detectors, num, *args, per_step=None, md=None):
+    # For rel_scan, num is the _last_ positional arg instead of the first one.
+    # Notice the swapped order here.
+    yield from rel_scan(detectors, *args, num, per_step=per_step, md=md)
+
+
+def rel_scan(detectors, *args, per_step=None, md=None):
```

Comment:
is it worth doing enough processing to accept `num` as a kwarg or the last positional?

Reply:
Hmm. Probably. Will add that.